Kickstarter-iOS/Views/Controllers/ChangePasswordViewController.swift

```diff
@@ -77,8 +77,7 @@ final class ChangePasswordViewController: UIViewController {
       |> UILabel.lens.text %~ { _ in Strings.Confirm_password() }
 
     _ = confirmNewPasswordTextField
-      |> passwordFieldStyle
-      |> settingsFormFieldStyle
+      |> settingsPasswordFormFieldStyle
```

Comment:
Also,  `settingsPasswordFormFieldStyle` inherits from `passwordFieldStyle`, that sets `secureTextEntry` to `true`. So it's fine to remove the `textAlignment` and `secureTextEntry` lines from all passwordTextFields of this view controller.